src/main/java/io/fabric8/elasticsearch/plugin/kibana/DocumentBuilder.java

```diff
@@ -60,20 +56,20 @@ public DocumentBuilder intervalName(String name) {
 	
 	public DocumentBuilder applicationFields(boolean use_cdm) {
 		if ( use_cdm )
-			return addField(FIELDS_FIELD, StringEscapeUtils.escapeJava(APPLICATION_CDM_FIELDS));
```

Comment:
Actually, these are what are called "Kibana index patterns".  If they are omitted, Kibana will see if the index/index pattern has index mappings defined and use them instead.